### ISSUE TITLE
Hidden title bar does not seem to work properly

### ISSUE THREAD

**Issue by DeFlight** [hawkeye-stan/msfs-popout-panel-manager#113]
Not sure if I am doing something wrong or there is a setting I am missing.  The Hidden title bar does nothing for me.  

Is there a setting on the computer I should be aware of to make this work?
![Screenshot 2024-03-24 121411](https://github.com/hawkeye-stan/msfs-popout-panel-manager/assets/9827913/60907532-f959-4c91-8e77-c97c0ab4d5cc)

**Comment by hawkeye-stan** [comment]
Hide title bar should work. Do you have any custom Windows theme installed on your machine that could interfere with the hide title bar function.

**Comment by DeFlight** [comment]
I have got it to work a little better now.  I had to just redo it and place them one at a time.  I also notice if I move the windows while hidden title bar is selected it does not work as well as if I uncheck title bar hidden, move the window, then check it.
On another note.  Not sure this is known but the software does not close.  I have had to go to the task manager and manually end the task.  Never used to have this issue so not sure it is new.  It also seems to hang MSFS from closing as well for some reason but I do not think they are related.
